### DISCUSSION TITLE
why do we need pinata?

### DISCUSSION THREAD

**Discussion by luahana** [smartcontractkit/full-blockchain-solidity-course-py#1254] [Q&A] [answered]
I guess I don't quite understand difference between pinata and ipfs.

Could someone explain what they are?

from my understanding ipfs is where we save data(photos and videos) decentralized just like cloud storage.

And I thought ipfs iis decentralized so it would not go down right?

then what is the need for pinata?

and what is filecoin differ from ipfs?

**Comment by cromewar** [comment, marked as answer]
Hello @luahana 
IPFS is just a system/protocol for decentralized storage, if you want to use just IPFS and you want your files to be maintained you must run your own node of it constantly, even when your computer is off, think about IPFS as the core technology
Pinata and Filecoin are services which uses IPFS as base but offers you the storage capabilities on their own nodes, so you don't have to run your own.

**Reply by luahana** [reply]
I see thanks!
